ASICBlocks/Formatter.py

```diff
@@ -229,7 +229,7 @@ def formatBestChoiceOutput(row, nTC = 1, debug=False):
 
     bx_cnt = row['BX_CNT']
     header =  format(bx_cnt, '#0%ib'%(7))[2:]
-    header = header[0] + header[2:]
+    header = header[1:]
```

Comment:
This will result in the wrong header, should be reverted to dropping bit 1, rather than bit 0

Reply:
Reverted.